src/components/views/rooms/RoomListPanel/RoomListCell.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+import React, { type JSX } from "react";
+import { type Room } from "matrix-js-sdk/src/matrix";
+
+import { _t } from "../../../../languageHandler";
+import { Flex } from "../../../utils/Flex";
+import DecoratedRoomAvatar from "../../avatars/DecoratedRoomAvatar";
+
+interface RoomListCellProps extends React.HTMLAttributes<HTMLButtonElement> {
+    /**
+     * The room to display
+     */
+    room: Room;
+}
+
+/**
+ * A cell in the room list
+ */
+export function RoomListCell({ room, ...props }: RoomListCellProps): JSX.Element {
```

Comment:
Would be nice if this component eventually had it's own vm.

Reply:
Hmm, it can be introduced when adding the hover menu I think. In all cases, this component needs the room in props so it will be an empty vm here. We can move the `Action.ViewRoom` dispatcher when will be created this vm.